cmd/ttn-lw-cli/commands/config.go

```diff
@@ -71,30 +72,43 @@ func (c Config) getHosts() []string {
 	return getHosts(hosts...)
 }
 
-// DefaultConfig contains the default config for the ttn-lw-cli binary.
-var DefaultConfig = Config{
-	Base: conf.Base{
-		Log: conf.Log{
-			Level: log.InfoLevel,
+// MakeDefaultConfig builds the default config for the ttn-lw-cli binary for a given host.
+func MakeDefaultConfig(clusterHost string, insecure bool) Config {
+	clusterGRPCAddress := fmt.Sprintf("%s:8884", clusterHost)
+	clusterHTTPAddress := fmt.Sprintf("https://%s:8885", clusterHost)
+	if insecure {
+		clusterGRPCAddress = fmt.Sprintf("%s:1884", clusterHost)
+		clusterHTTPAddress = fmt.Sprintf("http://%s:1885", clusterHost)
+	}
+
+	return Config{
+		Base: conf.Base{
+			Log: conf.Log{
+				Level: log.InfoLevel,
+			},
 		},
-	},
-	InputFormat:                        "json",
-	OutputFormat:                       "json",
-	OAuthServerAddress:                 clusterHTTPAddress + "/oauth",
-	IdentityServerGRPCAddress:          clusterGRPCAddress,
-	GatewayServerEnabled:               true,
-	GatewayServerGRPCAddress:           clusterGRPCAddress,
-	NetworkServerEnabled:               true,
-	NetworkServerGRPCAddress:           clusterGRPCAddress,
-	ApplicationServerEnabled:           true,
-	ApplicationServerGRPCAddress:       clusterGRPCAddress,
-	JoinServerEnabled:                  true,
-	JoinServerGRPCAddress:              clusterGRPCAddress,
-	DeviceTemplateConverterGRPCAddress: clusterGRPCAddress,
-	DeviceClaimingServerGRPCAddress:    clusterGRPCAddress,
-	QRCodeGeneratorGRPCAddress:         clusterGRPCAddress,
+		InputFormat:                        "json",
```

Comment:
We also want to set the `CredentialsID` so that we can be logged in to multiple deployments at the same time.

Reply:
The `credentials-id` is kept empty, as is currently the case. Added a `--credentials-id` flag that adds that id to the resulting config file.